src/app/public/modules/flyout/flyout.component.spec.ts

```diff
@@ -443,6 +508,47 @@ describe('Flyout component', () => {
     expect(mouseUpSpy).toHaveBeenCalled();
   }));
 
+  it('should send the resized width to the UI config service when a settings key is given', fakeAsync(() => {
+    openFlyout({ defaultWidth: 500, settingsKey: 'testKey' });
+    fixture.detectChanges();
+    tick();
+    spyOn(SkyFlyoutComponent.prototype, 'onMouseMove').and.callThrough();
+    spyOn(SkyFlyoutComponent.prototype, 'onHandleRelease').and.callThrough();
```

Comment:
Why are you spying on these two mouse events? I don't see you utilizing them anywhere...

Reply:
There were two tests like this due to copy pasta. I've fixed them both.